repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
That's scary! How would any issues ever be debugged?

With cassandra we generate a unique package version every build.  The project code is still `0.1.0-SNAPSHOT` but we still have a unique package version that we can use to trace back what is actually running.